package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "@lifeomic/axios-fetch",
-  "version": "1.5.2",
+  "version": "1.6.0",
```

Comment:
I'm always hesitant to make major refactors a minor bump, so I'd vote for 2.x but not strongly enough to block on it.

Reply:
Bumped to 2.0.0